src/pybind/mgr/dashboard/services/orchestrator.py

```diff
@@ -105,8 +109,8 @@ def reload(self, service_type, service_ids):
 
 class OsdManager(ResourceManager):
     @wait_api_result
-    def create(self, drive_group):
-        return self.api.create_osds(drive_group)
+    def create(self, drive_group_specs):
+        return self.api.apply_drivegroups(drive_group_specs)
```

Comment:
Relates to 

https://github.com/ceph/ceph/pull/34001/files#diff-e4aa6dad26f1313c41e1c44363d5a000R262

But it should work nevertheless